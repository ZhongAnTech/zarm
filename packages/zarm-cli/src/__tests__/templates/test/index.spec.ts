import testTemplate from '../../../templates/test';

describe('templates', () => {
  describe('test', () => {
    it('should create test template', () => {
      const template = testTemplate.indexTemp('MyTestComponent');
      expect(template).toMatchSnapshot();
    });
  });
});
