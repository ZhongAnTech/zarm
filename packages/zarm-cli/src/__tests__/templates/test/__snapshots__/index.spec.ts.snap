// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`templates test should create test template 1`] = `
"import React from 'react';
import { render } from 'enzyme';
import toJson from 'enzyme-to-json';
import MyTestComponent from '../index';

describe('MyTestComponent', () => {
  it('renders correctly', () => {
    const wrapper = render(
      <MyTestComponent />,
    );
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
"
`;
